packages/video_player/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,27 +1,36 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.flutter.plugins.videoplayerexample">
+  package="io.flutter.plugins.videoplayerexample">
 
-    <uses-permission android:name="android.permission.INTERNET"/>
+  <application
+    android:icon="@mipmap/ic_launcher"
+    android:label="video_player_example"
+    android:name="io.flutter.app.FlutterApplication"
+    android:networkSecurityConfig="@xml/network_security_config">
+    <activity
+      android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection"
+      android:exported="true"
+      android:hardwareAccelerated="true"
+      android:launchMode="singleTop"
+      android:name=".EmbeddingV1Activity"
+      android:theme="@style/LaunchTheme"
+      android:windowSoftInputMode="adjustResize">
+      <meta-data
+        android:name="io.flutter.app.android.SplashScreenUntilFirstFrame"
+        android:value="true"/>
+    </activity>
+    <activity
+      android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection"
+      android:hardwareAccelerated="true"
+      android:launchMode="singleTop"
```

Comment:
not necessary per Matt's suggestion. https://github.com/flutter/plugins/pull/2164#discussion_r334189140

Reply:
Done.